app/src/components/alert/SnackbarComponent.tsx

```diff
@@ -0,0 +1,44 @@
+import Alert from '@mui/material/Alert';
+import Snackbar from '@mui/material/Snackbar';
+import { useFormikContext } from 'formik';
+import { useEffect, useState } from 'react';
+
+const SnackbarComponent = () => {
+  const formikProps = useFormikContext<any>();
+  const { errors, submitCount } = formikProps;
+  const [openSnackbar, setOpenSnackbar] = useState({ open: false, msg: '' });
+
+  useEffect(() => {
+    const showSnackBar = (message: string) => {
```

Comment:
Could probably do away with this anonymous function and just directly call `setOpenSnackbar`
```javascript
useEffect(() => {
  if (!errors || submitCount <= 0) {
    return;
  }

  setOpenSnackbar({ open: true, message: 'Missing ...' });
}, [errors, submitCount]);
```